noir_stdlib/src/sha256.nr

```diff
@@ -132,64 +128,52 @@ fn digest(msg: [u8]) -> [u8; 32]
 
     // If i >= 57, there aren't enough bits in the current message block to accomplish this, so
     // the 1 and 0s fill up the current block, which we then compress accordingly.
-    if i >= 57 // Not enough bits (64) to store length. Fill up with zeros.
-    {
-        if i < 64
-        {
-            for _i in 57..64
-            {
-                if i <= 63
-                {
+    if i >= 57 { // Not enough bits (64) to store length. Fill up with zeros.
+        if i < 64 {
+            for _i in 57..64 {
+                if i <= 63 {
                     msg_block[i] = 0;
-                    i = i + 1;
+                    i += 1;
                 }
-            };
+            }
         }
         c = h;
         c = sha_c(msg_u8_to_u32(msg_block), c);
-        for j in 0..8
-        {
+        for j in 0..8 {
             h[j] = c[j] + h[j];
-        };
+        }
         
         i = 0;
     }
 
-    for _i in 0..64 // In any case, fill blocks up with zeros until the last 64 (i.e. until i = 56).
-    {
-        if i < 56
-        {
+    for _i in 0..64 {// In any case, fill blocks up with zeros until the last 64 (i.e. until i = 56).
+        if i < 56 {
             msg_block[i] = 0;
             i = i + 1;
-        } else if i < 64
-            {
-                let mut len = 8*crate::array::len(msg) as u64;
-                for j in 0..8
-                {
-                    msg_block[63 - j] = len as u8;
-                    len = len >> 8;
-                };
-                i = i + 8;
-            }
-    };
+        } else if i < 64 {
+            let mut len = 8 * msg.len() as u64;
+            for j in 0..8 {
+                msg_block[63 - j] = len as u8;
+                len = len >> 8;
+            };
+            i = i + 8;
```

Comment:
Note: can use += here -- only noting it as I saw you did it above. Can ignore if you find whats there to be better